src/main/csharp/HelloWorld/Program.cs

```diff
@@ -11,7 +11,7 @@
 # or implied.  See the License for the specific language governing permissions and limitations
 # under the License.
 #
-﻿// Copyright (c) YugaByte, Inc.
+﻿// Copyright (c) Yugabyte, Inc.
```

Comment:
This line seems redundant, given there's already a copyright on line 2 like every other file has...